Gemfile

```diff
@@ -58,12 +58,12 @@ gem "rack-utf8_sanitizer", "~> 1.6"
 gem "turnout", "~> 2.5"
 gem "audited", "~> 4.8"
 gem "git", "~> 1.5"
-gem "graphql", "~> 1.9.16"
```

Comment:
There are many gems updates, if we are reversing maybe it might be worth keeping the old ones. I remember there was in the pass a problem related to graphql gem update that affected the api and we couldn't see it.

Reply:
The specific update is to the latest version of the `graphql` gem. This is now finally possible, as the `apollo-federation` gem now supports `graphql` version 1.10.